amundsen_application/static/js/components/TableDetail/EditableSection/tests/index.spec.tsx

```diff
@@ -49,7 +50,7 @@ describe("EditableSection", () => {
     const { wrapper, props } = setup({ title: customTitle }, <TagInput/>);
 
     it("sets the title from a prop", () => {
-      expect(wrapper.find(".section-title").text()).toBe(customTitle);
+      expect(wrapper.find(".section-title").text()).toBe("Custom Title");
```

Comment:
Note, I am currently applying to all section-titles in EditableSection. Is this ok?

Reply:
Yes, that's the intent to have a title that looks consistent.